server/api/tasks/resources.py

```diff
@@ -381,10 +396,41 @@ def get(self, project_id):
                 description: Internal Server Error
         """
         try:
-            locked_tasks = ProjectService.get_task_for_logged_in_user(
-                project_id, tm.authenticated_user_id
+            order_by = request.args.get("orderBy")
+            status = request.args.get("status")
+
+            as_file = (
+                strtobool(request.args.get("as_file"))
+                if request.args.get("as_file")
+                else True
             )
-            return locked_tasks.to_primitive(), 200
+
+            if status:
+                try:
+                    status = TaskStatus[status.upper()].value
+                except KeyError:
+                    status = None
+
+            if order_by not in [
+                "building_area_diff",
```

Comment:
Can we use some more generic term here? I think for now these are buildings, but soon there will be roads or others. So what about using `effort_prediction_diff` or something like that?